.github/workflows/build.yml

```diff
@@ -20,9 +20,10 @@ name: Build & Test
 on: [push]
 
 jobs:
-  build-ubuntu:
-    name: build-ubuntu
+  build-linux:
+    name: build-linux
     runs-on: ubuntu-latest
```

Comment:
I think this `runs-on` line may be redundant based on this documentation here https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer, although I do see where `runs-on` is a required field so it looks like they are contradicting themselves in their own docs. Either way this looks a little confusing to me and I can't tell if we're using ubuntu or alpine.

Reply:
turns out this is required -- I guess you can run some steps in docker and others in the host (ubuntu in this case)